Formula/k/kubetail.rb

```diff
@@ -21,7 +21,7 @@ class Kubetail < Formula
 
   depends_on "go" => :build
   depends_on "make" => :build
-  depends_on "node" => :build
+  depends_on "node@22" => :build
```

Comment:
maybe also log an upstream issue for this?

Reply:
I checked upstream for an issue or PR, but found none.
Doesn't feel right to report an issue which isn't an issue yet, and node 22 will be supported longer than 23:

![image](https://github.com/user-attachments/assets/fef50f2f-54b2-4c29-a040-538f8d47d220)

Reply:
Wow, @amorey is already fixing it in upstream
* https://github.com/kubetail-org/kubetail/pull/139

supposedly coming in 0.0.7 release

Reply:
Tryying 0.0.7 with the fix in
* https://github.com/Homebrew/homebrew-core/pull/195866

Reply:
Thanks for the heads up about the problem. I patched the bug and published a new version (0.0.7) which should compile with Node 23. The source of the error was a tailwind config issue which was resolved by updating the config file format from commonjs to typescript.

Reply:
Thanks @amorey! 🍻